src/BoilerPlate/Game.cpp

```diff
@@ -384,9 +398,11 @@ void Game::ManageInput()
 		ResetLimiter();
 	}
 
-	if (inputManager.GetR())
+	if (inputManager.GetR() && inputLimiter == 0)
```

Comment:
What's `GetR`? Is it state for the `R` key? Does it take into account if it's lower or upper case? Is it evaluating the state of the ship going `right`? Think about naming for this.